### ISSUE TITLE
SVG Image Role (WCAG 2.2 Level A - Accessibility)

### ISSUE THREAD

**Issue by ryangrundy7** [ONSdigital/design-system#3118]
## Background

SRM had a DAC assessment done in January and it raised this issue

## Issue

Non-text content presented did not have a text alternative that served an equivalent purpose, as Aria had not been implemented correctly.

![Screenshot from 2024-04-02 14-13-16](https://github.com/ONSdigital/design-system/assets/35301276/689df50c-d013-4499-ac24-6432d876c087)

The lock SVG image hasn't been given a `role="img"` and a text alternative. Interpretation of inline SVG graphics can vary depending on the screen reader so should be made consistent.

Same goes for the chevron on the "Who can I contact for further information?"

![Screenshot from 2024-04-02 14-25-26](https://github.com/ONSdigital/design-system/assets/35301276/19f18dd5-b318-4fb2-925c-3411ed3f52b4)

The report does mention some that affects the eQ pages as well

## Info
[DAC Assessment Report](https://confluence.ons.gov.uk/pages/viewpage.action?spaceKey=SDC&title=DAC+assessment+report)
Issue ID: DAC_SVG_Image_Role_01
Pages: 13-15
